client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -347,7 +427,7 @@ class DomainDetailsForm extends Component {
 	}
 
 	handleSubmitButtonClick = ( event ) => {
-		event.preventDefault();
+		event && event.preventDefault();
```

Comment:
No, just vestigial. I was experimenting with passing pseudo submit events shortly before I threw my hands in the air and said "Reduxify!" :)